ksql-rest-app/src/test/resources/ksql-plan-schema/schema.json

```diff
@@ -0,0 +1,1062 @@
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "createStreamV1" ],
+          "default" : "createStreamV1"
+        },
+        "sqlExpression" : {
+          "type" : "string"
+        },
+        "sourceName" : {
+          "type" : "string"
+        },
+        "schema" : {
+          "type" : "string"
+        },
+        "keyField" : {
+          "type" : "string"
+        },
+        "timestampExtractionPolicy" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/MetadataTimestampExtractionPolicy"
+          }, {
+            "$ref" : "#/definitions/StringTimestampExtractionPolicy"
+          }, {
+            "$ref" : "#/definitions/LongColumnTimestampExtractionPolicy"
+          } ]
+        },
+        "serdeOptions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string",
+            "enum" : [ "UNWRAP_SINGLE_VALUES" ]
+          }
+        },
+        "topic" : {
+          "$ref" : "#/definitions/KsqlTopic"
+        }
+      },
+      "title" : "createStreamV1",
+      "required" : [ "@type", "sqlExpression", "sourceName", "schema", "keyField", "timestampExtractionPolicy", "serdeOptions", "topic" ]
+    },
+    "MetadataTimestampExtractionPolicy" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "metadatav1" ],
+          "default" : "metadatav1"
+        },
+        "timestampField" : {
+          "type" : "string"
+        }
+      },
+      "title" : "metadatav1",
+      "required" : [ "@type" ]
+    },
+    "StringTimestampExtractionPolicy" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "stringColumnv1" ],
+          "default" : "stringColumnv1"
+        },
+        "timestampField" : {
+          "type" : "string"
+        },
+        "format" : {
+          "type" : "string"
+        }
+      },
+      "title" : "stringColumnv1",
+      "required" : [ "@type", "timestampField" ]
+    },
+    "LongColumnTimestampExtractionPolicy" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "longColumnv1" ],
+          "default" : "longColumnv1"
+        },
+        "timestampField" : {
+          "type" : "string"
+        }
+      },
+      "title" : "longColumnv1",
+      "required" : [ "@type", "timestampField" ]
+    },
+    "KsqlTopic" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "kafkaTopicName" : {
+          "type" : "string"
+        },
+        "keyFormat" : {
+          "$ref" : "#/definitions/KeyFormat"
+        },
+        "valueFormat" : {
+          "$ref" : "#/definitions/FormatInfo"
+        },
+        "ksqlSink" : {
+          "type" : "boolean"
```

Comment:
definitely shouldn't be exposing this. (follow up PR to fix)

Reply:
https://github.com/confluentinc/ksql/pull/3911